### ISSUE TITLE
Exclude Certain Entities from Lighting

### ISSUE THREAD

**Issue by aMyTimed** [jgayfer/bevy_light_2d#31]
would it be possible to add component that excludes the entity it's on from being affected by lights?

**Comment by jgayfer** [comment]
Not possible currently, but I agree it's definitely needed.

Either some type of sprite specific light mask, or a way to use the z-index to order lights and sprites.

**Comment by jgayfer** [comment]
@amytimed I know you opened #44, which is a different issue, but it's worth pointing out at #49 could possibly address some of the sub-problems covered by this issue.

**Comment by sbesh91** [comment]
Is this exclude under current consideration for development?

**Comment by jgayfer** [comment]
> Is this exclude under current consideration for development?

Certainly! It's not something I'm working on, but it would be great to have some type of control over.
